remote.go

```diff
@@ -539,6 +556,26 @@ func (c *RemoteCollection) Create(name string, url string) (*Remote, error) {
 	return remote, nil
 }
 
+func (c *RemoteCollection) CreateOptions(url string, option *RemoteCreateOptions) (*Remote, error) {
```

Comment:
```suggestion
func (c *RemoteCollection) CreateWithOptions(url string, option *RemoteCreateOptions) (*Remote, error) {
```

for consistency with the `_with_` part of the C function.

also, can there be a docstring for this function?